src/qtt/algorithms/fitting.py

```diff
@@ -248,10 +249,11 @@ def gaussian_model(x, mean, sigma, amplitude):  # type: ignore
             y = gaussian(x, mean, sigma, amplitude)
             return y
 
+    lmfit_method = 'least_squares'
     lmfit_model = Model(gaussian_model)
     lmfit_model.set_param_hint('amplitude', min=0)
     lmfit_result = lmfit_model.fit(
-        y_data, x=x_data, **dict(zip(lmfit_model.param_names, initial_parameters)), verbose=verbose)
+        y_data, x=x_data, **dict(zip(lmfit_model.param_names, initial_parameters)), verbose=verbose, method=lmfit_method)
```

Comment:
Maybe add to the docstring that 'least_squares' is used as the fitting-method instead of the default 'leastsq'

Reply:
Updated the docstrings